network-api/networkapi/wagtailpages/pagemodels/buyersguide/products.py

```diff
@@ -292,26 +293,44 @@ class ProductPageEvaluation(models.Model):
 
     @property
     def total_votes(self):
-        return self.votes.count()
+        try:
+            return self._total_votes
+        except AttributeError:
+            return self.votes.count()
 
     @property
     def total_creepiness(self):
-        return sum([vote.value for vote in self.votes.all()])
+        try:
+            return self._total_creepiness
+        except AttributeError:
+            return sum([vote.value for vote in self.votes.all()])
```

Comment:
**suggestion:** Should this throw an exception rather than falling back to a manual calculation? This would prevent someone from accidentally loading all those votes in to memory without realising it.

Reply:
That's a great point, thanks! I suppose we can shoot ourselves in the foot now without noticing it! I'll remove this and throw an exception